### PR TITLE
Changes tests to account for changed comment format.

### PR DESCRIPTION
**Pull request by ellisonmarks**
We now put multiple slices into a single comment, so we have to do a bit different comment parsing.

### CODE REVIEW

File: spec/lib/tacos_demo.py [hunk 1]
```diff
@@ -213,25 +213,15 @@ def parse_comments(
 def parse_comment(
```
Comment (lynnagara) [inline spec/lib/tacos_demo.py:213]: can you write a docstring with an example of the transformation that is happening here, it's seems quite complex and is not easy to read

File: spec/lib/tacos_demo.py [hunk 2]
```diff
@@ -213,25 +213,15 @@ def parse_comments(
...
-        if not tag_start:
-            return
-
-        tag, tag_end, remainder = remainder.partition(COMMENT_TAG_END)
-
-        tag = Parse(tag)
-        job = tag.before.first("(")  # )
+    lines = comment.splitlines(keepends=True)
+    points = [i for i, l in enumerate(lines) if COMMENT_TAG in l]
+    points.append(len(lines))
```
Comment (lynnagara) [inline spec/lib/tacos_demo.py:218]: what does points represent exactly? is that the indexes with the total line count appended on the list?

File: spec/lib/tacos_demo.py [hunk 3]
```diff
@@ -213,25 +213,15 @@ def parse_comments(
...
+    chunks = ["".join(lines[i:j]) for i, j in pairwise(points)]
+    for chunk in chunks:
+        before, _tag_start, after = chunk.partition(COMMENT_TAG)
+        job, _tag_end, after = after.partition(COMMENT_TAG_END)
```
Comment (lynnagara) [inline spec/lib/tacos_demo.py:222]: doesn't the comment_tag always appear at the start and the comment_tag_end appear at the end? this `partition` function seems to imply it's in the middle

File: spec/lib/tacos_demo.py [hunk 4]
```diff
@@ -213,25 +213,15 @@ def parse_comments(
...
-        comment = "".join((comment, tag_start, tag, tag_end))
-
-        parsed = tag.between("(", ")")
-        if parsed == tag:
-            slice = Slice(".")
-        else:
-            slice = Slice(parsed)
-            slice = slice.relative_to(slices_subpath)
-        yield job, slice, comment
+        slice = Slice(before[before.index("terraform/") : -1])
```
Comment (lynnagara) [inline spec/lib/tacos_demo.py:225]: does this mean `terraform` has to be part of the path now?

File: spec/lib/tacos_demo.py (hunk 2 above)
Comment (ellisonmarks) [reply spec/lib/tacos_demo.py:218]: points is the delimiting indexes of the individual sections in a combined comment.

File: spec/lib/tacos_demo.py (hunk 3 above)
Comment (ellisonmarks) [reply spec/lib/tacos_demo.py:222]: it appears in the middle of the first line of a chunk. the goal of these two lines is to get the bit between the start and end tags.

File: spec/lib/tacos_demo.py (hunk 4 above)
Comment (ellisonmarks) [reply spec/lib/tacos_demo.py:225]: yes, which I think is always the case for the tests. this was sort of desperation, as there's not a reliable delimiter on the left side of the slice. may need to think on this part more.

File: spec/lib/tacos_demo.py (hunk 1 above)
Comment (bukzor) [reply spec/lib/tacos_demo.py:213]: a unit test is a good match for that kind of situation, too

File: spec/lib/tacos_demo.py (hunk 4 above)
Comment (bukzor) [reply spec/lib/tacos_demo.py:225]: There's reliably a space on the left side.

Comment (ellisonmarks) [reply spec/lib/tacos_demo.py:230]: seems like there's not when it's a link in markdown?

**Review by lynnagara — APPROVED** [review]
Let's get this fix in.

Curious if you considered using regex for the comment parsing? It seems like it might be a good fit.

**Pull request merged**